src/client.ts

```diff
@@ -199,14 +199,25 @@ class Client extends EventEmitter implements IClient {
         const { toggles } = metrics.bucket;
 
         Object.keys(toggles).forEach((toggleName) => {
-            const yesCount: number = toggles[toggleName].yes ?? 0;
-            const noCount: number = toggles[toggleName].no ?? 0;
+            const toggle = toggles[toggleName];
+            const yesCount: number = toggle.yes ?? 0;
+            const noCount: number = toggle.no ?? 0;
             [...Array(yesCount)].forEach(() =>
                 this.metrics.count(toggleName, true),
             );
             [...Array(noCount)].forEach(() =>
                 this.metrics.count(toggleName, false),
             );
+            const variants = toggle.variants;
+            if (variants) {
+                Object.entries(variants).forEach(
+                    ([variantName, variantCount]) => {
+                        [...Array(variantCount)].forEach(() =>
+                            this.metrics.countVariant(toggleName, variantName),
+                        );
```

Comment:
Oh, interesting. Is there no way to say count `n` variants? Or is that part of the planned refactor?

Reply:
There's only single count in the metrics. It's the same as regular count though. The tests will be added after a refactor as currently metrics are not injectable and difficult to test.

Reply:
Got it. I'm not really worried about it, but if you have a thousand variants on a thousand features, each with a thousand hits, then that'll take some time to chew through. Do we have any use cases like that or any cases where it might cause issues? It's not like this is a super hot path, anyway, right?